src/lib.rs

```diff
@@ -1,5 +1,6 @@
 #![cfg_attr(nightly, feature(doc_cfg))]
 #![deny(missing_docs)]
+#![forbid(unsafe_code)]
```

Comment:
Is there any way this could backfire?

Reply:
if this crate is going to implement some unsafe code in the future, then yes... but there should be NO motiv behind implementing unsafe code